modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/tree/io/BplusIo.java

```diff
@@ -346,14 +357,15 @@ public abstract void copyItems(
             L row,
             @Nullable byte[] rowBytes,
             long rightId,
-            boolean needRowBytes
+            boolean needRowBytes,
+            int partId
     ) throws IgniteInternalCheckedException {
         assertPageType(pageAddr);
 
         int cnt = getCount(pageAddr);
 
         // Move right all the greater elements to make a free slot for a new row link.
-        copyItems(pageAddr, pageAddr, idx, idx + 1, cnt - idx, false);
+        copyItems(pageAddr, pageAddr, idx, idx + 1, cnt - idx, false, partId);
```

Comment:
`partId` doesn't seem necessary for copying, why is it here?

Reply:
Fix it